nuget-tools.cake

```diff
@@ -0,0 +1,173 @@
+var NugetUrl = "https://api.nuget.org/v3/index.json";
+
+// UWP IL2CPP dependencies.
+var UwpIL2CPPDependencies = new [] {
+   new NugetDependency("SQLitePCLRaw.bundle_green", "2.0.2", ".NETStandard,Version=v2.0"),
+   new NugetDependency("Newtonsoft.Json", "12.0.3", ".NETStandard,Version=v2.0"),
+   new NugetDependency("SQLitePCLRaw.provider.e_sqlite3", "2.0.2", ".NETStandard,Version=v2.0"),
+   new NugetDependency("SQLitePCLRaw.core", "2.0.2", ".NETStandard,Version=v2.0"),
+   new NugetDependency("SQLitePCLRaw.lib.e_sqlite3", "2.0.2", "UAP10.0")
+};
+
+var UwpIL2CPPJsonUrl = SdkStorageUrl + "Newtonsoft.Json.dll";
```

Comment:
Why do you need both NugetDependency and UwpIL2CPPJsonUrl for it?

Reply:
Because removing this is addressed in another PR.